src/main/java/org/sopt/lequuServer/domain/member/dto/response/MypageNoteListResponseDto.java

```diff
@@ -0,0 +1,32 @@
+package org.sopt.lequuServer.domain.member.dto.response;
+
+import org.sopt.lequuServer.domain.note.model.Note;
+
+import java.time.format.DateTimeFormatter;
+
+public record MypageNoteListResponseDto(
+        String bookUuid,
+        String title,
+        Long noteId,
+        String content,
+        String noteDate,
+
+        int noteBackgroundColor,
+        String noteBackgroundImage
+) {
+    public static MypageNoteListResponseDto of(Note note) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy.MM.dd");
+        String noteDate = note.getCreatedAt().format(formatter);
+
+        String background = note.getBackground();
+        if (background.endsWith(".jpg")) {
+            return new MypageNoteListResponseDto(note.getBook().getUuid(), note.getBook().getTitle(),
+                    note.getId(), note.getContent(),
+                    noteDate, -1, background);
+        } else {
+            return new MypageNoteListResponseDto(note.getBook().getUuid(), note.getBook().getTitle(),
+                    note.getId(), note.getContent(),
+                    noteDate, Integer.parseInt(background), "");
+        }
```

Comment:
[P2] 클린코드를 위해서는 else 예약어 사용을 지양하는 편이 좋다고 합니다!
위 코드는
```suggestion
        if (background.endsWith(".jpg")) {
            return new MypageNoteListResponseDto(note.getBook().getUuid(), note.getBook().getTitle(),
                    note.getId(), note.getContent(),
                    noteDate, -1, background);
        }
        return new MypageNoteListResponseDto(note.getBook().getUuid(), note.getBook().getTitle(),
                note.getId(), note.getContent(),
                noteDate, Integer.parseInt(background), "");
```
이렇게 바꿀 수 있겠네요 :)